docs/vmalert-tool.md

```diff
@@ -0,0 +1,253 @@
+---
+sort: 11
+weight: 11
+menu:
+  docs:
+    parent: 'victoriametrics'
+    weight: 11
```

Comment:
It's need to be fixed, I think it should be placed between vmanomaly and case studies.
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/39937150/4367e648-8215-42ed-8804-48094d8ad514)
So I'm thinking set `Case studies and talks` bgain with "weight: 21" to leave first 20 to tools) 
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/466075dc35284be59e4026b1979cbcf1915aa20d/docs/CaseStudies.md?plain=1#L8
wdyt? @hagen1778

Reply:
Yes, it is ok to move it as it doesn't affect URL. While we're not changing URLs - it is ok shuffle items in menu block